Tests/RouterMultipleTests.swift

```diff
@@ -0,0 +1,301 @@
+//
+//  RouterMultipleTests.swift
+//  Kakapo
+//
+//  Copyright © 2018 devlucky. All rights reserved.
+//
+
+import Foundation
+import Quick
+import Nimble
+
+@testable import Kakapo
+
+class RouterMultipleTests: QuickSpec {
```

Comment:
Aren't these specs already covered in the Router specs?

Reply:
They were, I haven't added `new` specs, I just splat the existing one into multiple files to simplify the reading and to prevent **SwiftLint** from `failing` the build.

But if you prefer this to come from another PR, I think I should be able to make it happen.